### PR TITLE
Fixes #39 adding new script for webroot

### LINKED ISSUE #39: Add certbot_create_method 'webroot' for zero-downtime cert generation
After the work in #12 and #38, this role supports using Certbot's `--standalone` server for generating certs, which is great and perfectly adequate if you're either:

  1. Generating the cert on the first bootstrap of a server (since it's not causing an outage, it's just prepping for the first webserver startup).
  2. Okay with a few seconds of webserver downtime when you generate a new cert (note that there's no downtime during renewals).

However, some of the examples in #12's comments showed how certbot's `--webroot` option might be used with either Apache or Nginx to allow running webservers to not have to be stopped before a cert is generated.

Examples:

  - https://github.com/geerlingguy/ansible-role-certbot/issues/12#issuecomment-302216046
  - https://github.com/geerlingguy/ansible-role-certbot/issues/12#issuecomment-264167972 (specific to Apache)

See docs: https://certbot.eff.org/docs/using.html#webroot

### LINKED ISSUE #41: Support deployhook
Add deployhook support

### LINKED ISSUE #42: Add test cert support
add support for using non prod CA

### PR DESCRIPTION
**Pull request by rfaircloth-splunk**
Fixes #41 support deployhook (webroot only)
Fixes #42  support for test CA
Fixes #39 Support for webroot

### CODE REVIEW

File: tasks/create-cert-webroot.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+- name: Create deploy hook
+  copy:
+    content: "{{ certbot_deployhook }}"
+    dest: /etc/letsencrypt/renewal-hooks/deploy/ansible.sh
+    mode: u+rwx
+  run_once: yes
+  when: certbot_deployhook is defined
+
+- debug:
```
Comment (bensallen) [inline tasks/create-cert-webroot.yml]: Left over debug task?

Comment (kristianronningen) [reply tasks/create-cert-webroot.yml]: @rfaircloth-splunk Would indeed be nice to get this merged some time soon!

Comment (rfaircloth-splunk) [reply tasks/create-cert-webroot.yml]: Removed the debug statements

**Comment by simonspa** [comment]
@rfaircloth-splunk is there any chance you could rebase and document this? It would be a great addition to the role.

**Comment by rfaircloth-splunk** [comment]
I actually abandoned this for my usage for k8s

**Comment by simonspa** [comment]
I rebased it and added the missing documentation in #97

**Pull request closed without merging**